asyncapi/salesforce/Ballerina.toml

```diff
@@ -4,7 +4,7 @@ keywords = ["Sales & CRM/Customer Relationship Management", "Cost/Freemium"]
 org = "ballerinax"
 name = "trigger.salesforce"
 repository = "https://github.com/ballerina-platform/asyncapi-triggers"
-version = "0.9.0"
+version = "0.9.1"
```

Comment:
Shall we bump the version to 0.10.0 as there are considerable number of changes are added

Reply:
Done